### PR TITLE
Bump internal package version

### PR DESCRIPTION
**Pull request by guominrui**
### Description of the changes:
Bump MinVersion of appx manifest in release pipeline.

### How changes were validated:
Manually tested.

### CODE REVIEW

File: build/pipelines/azure-pipelines.ci-internal.yaml [hunk 1]
```diff
@@ -20,16 +20,24 @@ jobs:
...
   parameters:
     platform: x64
     runsettingsFileName: CalculatorUITests.release.runsettings
 
+- template: ./templates/run-ui-tests.yaml
+  parameters:
+    platform: x86
+    runsettingsFileName: CalculatorUITests.release.runsettings
+    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
+
```
Comment (guominrui) [inline build/pipelines/azure-pipelines.ci-internal.yaml:40]: Run all the UI tests as azure-pipelines.release.yaml previously did. After ci-internal passes the build and UI test, then trigger release pipeline, and UI test will not run in release pipeline.

File: build/pipelines/azure-pipelines.release.yaml [hunk 2]
```diff
@@ -26,27 +19,20 @@ jobs:
 - template: ./templates/build-app-internal.yaml
   parameters:
     platform: x64
+    isPublicRelease: true
```
Comment (guominrui) [inline build/pipelines/azure-pipelines.release.yaml:22]: Only bump MiniVersion for release.

File: build/pipelines/azure-pipelines.release.yaml [hunk 3]
```diff
@@ -4,13 +4,6 @@
 # Store and the Windows image. This pipeline relies on Microsoft-internal resources to run.
 #
 
-schedules:
-- cron: "0 7 * * *"
-  displayName: Daily midnight build
-  branches:
-    include:
-    - master
-
```
Comment (guominrui) [inline build/pipelines/azure-pipelines.release.yaml:13]: Release pipeline should be triggered after ci-internal done, and all build/tests passed successfully, so disable the auto trigger here.

Comment (hanzhang54) [reply build/pipelines/azure-pipelines.release.yaml:13]: It seems to be not necessary to remove this schedule. This schedule is for the master branch, and we should check the ci result before merge any PR into master.

File: build/pipelines/azure-pipelines.ci-internal.yaml [hunk 4]
```diff
@@ -20,16 +20,24 @@ jobs:
 - template: ./templates/build-app-internal.yaml
   parameters:
     platform: x86
+    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
```
Comment (hanzhang54) [inline build/pipelines/azure-pipelines.ci-internal.yaml:23]: We should also run the ci internal pipeline for pull request reason. The condition in build-app-internal.yaml will use the non-release version for build and pass the UI test as well.

**Pull request closed without merging**